server/paymentProviders/transferwise/index.ts

```diff
@@ -96,31 +101,45 @@ async function quoteExpense(
   }
 
   expense.collective = expense.collective || (await models.Collective.findByPk(expense.CollectiveId));
+  expense.host = expense.host || (await expense.collective.getHostCollective());
   const hasMultiCurrency = expense.currency !== expense.collective.currency;
+  const targetCurrency = payoutMethod.unfilteredData.currency as string;
   const quoteParams = {
     profileId: connectedAccount.data.id,
-    sourceCurrency: expense.collective.currency,
-    targetCurrency: <string>payoutMethod.unfilteredData.currency,
+    sourceCurrency: expense.host.currency,
+    targetCurrency,
     targetAccount,
   };
 
-  // Adapt the fees payer
-  if (expense.feesPayer === 'PAYEE') {
-    // We assume that expense.currency is always collective.currency
-    quoteParams['sourceAmount'] = expense.amount / 100;
-    if (hasMultiCurrency) {
-      throw new Error('Putting the fees on the payee is not yet supported for multi-currency expenses');
-    }
-  } else {
-    // Guarantees the target amount if in the same currency of expense
-    const { rate } = await getTemporaryQuote(connectedAccount, payoutMethod, expense);
-    quoteParams['targetAmount'] = (expense.amount / 100) * rate;
-  }
-
-  // Adapt for multi-currency
   if (hasMultiCurrency) {
+    assert(
+      expense.collective.currency === expense.host.currency,
+      'For multi-currency expenses, the host currency must be the same as the collective currency',
+    );
+    assert(
+      expense.currency === payoutMethod.unfilteredData.currency,
+      'For multi-currency expenses, the payout currency must be the same as the expense currency',
+    );
     quoteParams['targetCurrency'] = expense.currency;
     quoteParams['targetAmount'] = expense.amount / 100;
+  } else if (expense.feesPayer === 'PAYEE') {
+    assert(
```

Comment:
I would add a comment there just to make things clear for peeps who don't have the context

```suggestion
    // Using "else if" because customizing the fee payer is not allowed for multi-currency expenses. See `getCanCustomizeFeesPayer`.
    assert(
```

Reply:
That's a good idea. This `else-if` is definitely a logic simplification.